sqlnexus/PerfStatsAnalysis.sql

```diff
@@ -1739,6 +1743,11 @@ begin
 
 	print 'Add index to Table - dbo.CounterData || Column CounterDateTime ....'
 
+	create index [INDX_CounterDateTime]
```

Comment:
Thank you. Creating this index based on your suggestion and giving you credit as author
See this [commit ](https://github.com/microsoft/SqlNexus/commit/f8f96e87e989c838cf555e5bb2f8904513997e40)in a branch